handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -487,12 +491,36 @@ public final long sslCtxPointer() {
         return ctx;
     }
 
+    /**
+     * Enables OCSP stapling or throws an {@link Exception} if OCSP is not supported.
+     */
+    @UnstableApi
+    public final void enableOcsp() {
```

Comment:
is the intention to dynamically toggle this value? For example is it sufficient to just add a method to the `SslContextBuilder` and configure it in the constructor of this method.

For example isn't there some assumptions from the `SSL`'s perspective that the `SSL_CTX` has some callback set? We may be OK if we only ever allow a transition from disabled -> enabled, but it may introduce undefined behavior if we forget this relationship and add a enabled -> disabled transition.